src/diffusers/pipelines/kandinsky/pipeline_kandinsky.py

```diff
@@ -44,6 +45,21 @@ def get_new_h_w(h, w):
         new_w += 1
     return new_h * 8, new_w * 8
 
+def process_images(batch):
```

Comment:
we don't need this function, let's do post-processing similar to what's done here https://github.com/huggingface/diffusers/blob/kandinsky/src/diffusers/pipelines/kandinsky/pipeline_kandinsky_inpaint.py#L452